File: app/src/main/java/no/nav/sosialhjelp/soknad/consumer/exceptions/TjenesteUtilgjengeligException.java
package no.nav.sosialhjelp.soknad.consumer.exceptions;

import no.nav.sosialhjelp.soknad.domain.model.exception.SosialhjelpSoknadApiException;

public class TjenesteUtilgjengeligException extends SosialhjelpSoknadApiException {
    public TjenesteUtilgjengeligException(String message, Exception exception) {
        super(message, exception);
    }
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/client/exceptions/TjenesteUtilgjengeligException.kt
package no.nav.sosialhjelp.soknad.client.exceptions

import no.nav.sosialhjelp.soknad.common.exceptions.SosialhjelpSoknadApiException

class TjenesteUtilgjengeligException(message: String, throwable: Throwable?) :
    SosialhjelpSoknadApiException(message, throwable)
